src/test/java/com/g4s8/ghman/data/PgUserITCase.java

```diff
@@ -29,14 +29,20 @@
  *  methods of {@link com.g4s8.ghman.data.PgUser} class. Do not forget about
  *  all possible exceptions.
  */
-@ExtendWith(DatabaseExtension.class)
 final class PgUserITCase {
 
+    /**
+     * Database.
+     * @checkstyle VisibilityModifierCheck (5 lines)
```

Comment:
@olenagerasimova why is this needed?

Reply:
@olenagerasimova you missed this question I think

Reply:
@victornoel because checkstyle is complaining and I cannot make this field private

Reply:
@olenagerasimova I think you can at least make the field final

Reply:
@victornoel done